exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/utils/model_retrieval.ipynb

```diff
@@ -36,28 +36,30 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def load_huggingface_model_udf(model_name: str, conf: \"Secrets\") -> None:\n",
+    "def load_huggingface_model_udf(model_name: str) -> None:\n",
     "    \"\"\"\n",
     "    Loads specified model into BucketFS using a UDF provided with the Transformer Extension.\n",
+    "    There must be a global variable - sb_config - providing access to the secret store holding\n",
+    "    the configuration data.\n",
     "\n",
     "    model_name     - Name of the model at Huggingface hub, e.g. facebook/nllb-moe-54b.\n",
-    "    conf           - Sandbox configuration object.\n",
     "    \"\"\"\n",
     "\n",
     "    from exasol.connections import open_pyexasol_connection\n",
     "    from exasol.language_container_activation import get_activation_sql\n",
+    "    from exasol.ai_lab_config import AILabConfig as Cfg\n",
     "\n",
     "    sql = f\"\"\"\n",
-    "    SELECT {conf.SCHEMA}.TE_MODEL_DOWNLOADER_UDF(\n",
+    "    SELECT {sb_config.get(Cfg.db_schema)}.TE_MODEL_DOWNLOADER_UDF(\n",
     "        '{model_name}',\n",
-    "        '{conf.TE_MODELS_BFS_DIR}',\n",
-    "        '{conf.TE_BFS_CONN}',\n",
-    "        '{conf.TE_TOKEN_CONN}'\n",
+    "        '{sb_config.get(Cfg.te_models_bfs_dir)}',\n",
+    "        '{sb_config.get(Cfg.te_bfs_connection)}',\n",
+    "        '{sb_config.get(Cfg.te_hf_connection)}'\n",
```

Comment:
For my taste, there are too many things named "config" (to be precise: "Cfg" and "sb_config").
I apologize as this probably triggers a pathological spot in me :) 
I once worked for a project where each and everything was a "config".

Could you maybe think of more specific and at best more domain-related terms?

If we want to anticipate one of the findings from management review (Rename “Access Configuration” to "Open Secure Configuration Storage") then maybe "secrete configuration store" or "scs" could be an option (for `sb_config`). (I admit that "c" also refers to "configuration", sigh).

For `Cfg` we could use "Exasol AI Lab keys" or an abbreviation?
I just saw the class is already named `AILabConfig`. Maybe `alci` (short for "AI Lab configuration item")?

Reply:
I use an alias CKey in the notebook-connector (stands for Configuration Key). AILabConfig is basically a list of keys in the secret store. I can use the same alias in the notebooks instead of Cfg. Will it make it look better?